Only the GTK 4.x emoji data is native endian

Ref https://gitlab.gnome.org/World/Phosh/stevia/-/commit/62591bd120d36eb7edec8d1b8f904aac65bd1c5c
Ref https://gitlab.gnome.org/GNOME/gtk/-/commit/52cdf3056d86c7dd46517bcf37cf36dd511d4252

--- a/src/pos-emoji-db.c
+++ b/src/pos-emoji-db.c
@@ -134,7 +134,7 @@ pos_emoji_db_match_by_name (PosEmojiDb *self, const char *match, guint max_match
 
         g_variant_get_child (codes, i, "u", &code);
         if (code)
-          p += g_unichar_to_utf8 (code, p);
+          p += g_unichar_to_utf8 (GINT32_FROM_LE (code), p);
       }
       p += g_unichar_to_utf8 (0xFE0F, p); /* U+FE0F is the Emoji variation selector */
       *p = '\0';
